dev.js

```diff
@@ -0,0 +1,28 @@
+const Bundler = require('parcel-bundler');
+const Path = require('path');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+bundle().then(() => {
+  runElectron();
+});
+
+async function bundle() {
```

Comment:
rename to `bundleApp` and have it call two methods: `bundleFrontEnd` and `bundleBackend`?

If code duplication appears in `bundleFrontEnd` and `bundleBackend`, have them call a `bundle` function with proper arguments.